pyvelox/test/test_substrait.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pyvelox.pyvelox as pv
+import unittest
+import shutil, tempfile
+import os
+
+# Run the tests from the base path of the project
+BASE_PATH = os.path.join(os.getcwd(), "velox/substrait/tests/data/")
+
+
+class TestVeloxSubstrait(unittest.TestCase):
+    def setUp(self):
+        # create a temporary directory
+        pv.initialize_substrait()
+        self.test_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        # remove the temporary directory
+        pv.finalize_substrait()
+        shutil.rmtree(self.test_dir)
+
+    def test_simple(self):
+        other_path = BASE_PATH + "substrait_virtualTable.json"
```

Comment:
Is this something thats already checked in ?

Reply:
Yes indeed. I was merely trying to re-use what we already have an introduced 0 amount of text.